### DISCUSSION TITLE
Migrate not work some time

### DISCUSSION THREAD

**Discussion by AkhmedovSH** [sequelize/umzug#479] [Q&A]
v3.x (beta version)
SequelizeStorage
node 14.16.0
Vanilla commonjs
Electronjs 12.0.5
Work in dev mode after packaging the project have a problem below

logger.console shows me in dev mode
{ event: 'migrating', name: '00_initial.js' }
[{ event: 'migrated', name: '00_initial.js', durationSeconds: 0.195 }

logger.console shows me in prod mode
{ event: 'migrating', name: '00_initial.js' }
and stop process which mean as i think it starts migration but can not end and not return migrated status to console what can be possible problem?

**Comment by mmkal** [comment]
@AkhmedovSH most likely the migration you're running is hanging for some reason. If you can reproduce this in a minimal example with some specific migrations it can be looked into.

**Comment by AkhmedovSH** [comment]
> @AkhmedovSH most likely the migration you're running is hanging for some reason. If you can reproduce this in a minimal example with some specific migrations it can be looked into.

i take from example
const { Sequelize } = require('sequelize');

async function up({ context: queryInterface }) {
	await queryInterface.createTable('users', {
		id: {
			type: Sequelize.INTEGER,
			allowNull: false,
			primaryKey: true
		},
		name: {
			type: Sequelize.STRING,
			allowNull: false
		},
		createdAt: {
			type: Sequelize.DATE,
			allowNull: false
		},
		updatedAt: {
			type: Sequelize.DATE,
			allowNull: false
		}
	});
}

async function down({ context: queryInterface }) {
	await queryInterface.dropTable('users');
}

module.exports = { up, down };


can this package show concrete error in log it shows only which process run and sql errors but i want to find this process error maybe some if else try catch with console output in up down process i do not know

**Comment by mmkal** [comment]
There could be many reasons this would happen. Given it's working in development but not production, maybe there's a networking issue so the server can't reach the database in production. I'd suggest adding some log statements, and maybe using `Promise.race` to catch connection timeouts.
